components/dashboard/src/start/StartWorkspace.tsx

```diff
@@ -799,9 +799,25 @@ function ImageBuildView(props: ImageBuildViewProps) {
                 <WorkspaceLogs logsEmitter={logsEmitter} errorMessage={props.error?.message} />
             </Suspense>
             {!!props.onStartWithDefaultImage && (
-                <button className="mt-6 secondary" onClick={props.onStartWithDefaultImage}>
-                    Continue with Default Image
-                </button>
+                <>
+                    <div className="mt-6 w-11/12 lg:w-3/5">
+                        <p className="text-center text-gray-400 dark:text-gray-500">
+                            💡 You can use <code>gp rebuild</code> to validate and debug your workspace configuration
+                            &nbsp;&middot;&nbsp;
```

Comment:
**suggestion**: The middot character is only used so far when separating entities. For the help links we're always using a simple space character between the description and the help link, plus without a period character at the end.

```suggestion
                            💡 You can use <code>gp rebuild</code> to validate and debug your workspace configuration. 
```

For example, see screenshots below:

| Editor | SSH Keys | ... |
|-|-|-|
| <img width="438" alt="Screenshot 2023-01-17 at 7 11 19 PM" src="https://user-images.githubusercontent.com/120486/212966094-db6d64e7-2d7a-4806-944c-f5526aca9af3.png"> | <img width="348" alt="Screenshot 2023-01-17 at 7 11 28 PM" src="https://user-images.githubusercontent.com/120486/212966125-6914a721-d490-4d0d-a532-7f9413ce5f4c.png"> | ... |